src/resources/MachineLearning/ModelDetailedInfo/ModelDetailedInfoInterfaces.ts

```diff
@@ -96,4 +96,8 @@ export interface ModelWithDetails {
      * The current status of the model.
      */
     modelStatusInfo: ModelStatusInfo;
+    /**
+     * Specifies whether the model must be associated with a query pipeline to be effective.
+     */
+    requiresAssociation: number;
```

Comment:
Shouldn't that be a boolean?

Reply:
yep I don't know where I was looking lol - updated